uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/ManagerConf.java

```diff
@@ -47,14 +47,22 @@
   private static final String ENV = "manager.environment";
   private static final String MANAGER_INSTANCE_ID = "manager.instance.id";
 
-  private static final String CONTROLLER_PORT = "controller.port";
-
-  private static final String GRAPHITE_HOST = "controller.graphite.host";
-  private static final String GRAPHITE_PORT = "controller.graphite.port";
+  private static final String GRAPHITE_HOST = "manager.graphite.host";
+  private static final String GRAPHITE_PORT = "manager.graphite.port";
 
   private static final String METRICS_PREFIX = "controller.metrics.prefix";
   private static final String DEFAULT_METRICS_PREFIX = "ureplicator2-manager";
 
+  //graphiteReportFreqSec
```

Comment:
ManagerConf and ControllerConf have many common configurations, shall we abstract the common part?

Reply:
it's really a problem.
it's better to compress the configs in some config files